tests/conftest.py

```diff
@@ -3,13 +3,21 @@
 
 def pytest_configure(config):
     config.addinivalue_line("markers", "cuda_test: mark test as cuda_test to skip")
+    config.addinivalue_line("markers", "non_cuda_test: mark test as non_cuda_test to skip")
 
 
 def pytest_collection_modifyitems(items):
     if os.environ["TESTING_CONFIGURATION"] in ["CUDA_DIND_MARQO_OS"]:
-        return
-    skip_cuda_test = pytest.mark.skip(reason="need to setEnvVars "
-                                             "'TESTING_CONFIGURATION=CUDA_DIND_MARQO_OS' to run")
-    for item in items:
-        if "cuda_test" in item.keywords:
-            item.add_marker(skip_cuda_test)
\ No newline at end of file
+        # Skip non-cuda-tests if the env is CUDA
```

Comment:
we actually want the non cuda tests to run in a CUDA environment (because they should still still work as expected) 

Maybe make create a new marker like "ONLY_RUN_IN_CUDA" for this type of logic

Reply:
`non_cuda_test` means tests to run only when there is no cuda available. Should I change it to `cpu_only_test` instead?